code/controllers/stonedmc/subsystem/ticker.dm

```diff
@@ -17,25 +17,24 @@ SUBSYSTEM_DEF(ticker)
 	var/login_music							//Music played in pregame lobby
 
 	var/list/datum/mind/minds = list()		//The characters in the game. Used for objective tracking.
-	var/datum/mind/liaison
 
 	var/delay_end = FALSE					//If set true, the round will not restart on it's own
 	var/admin_delay_notice = ""				//A message to display to anyone who tries to restart the world after a delay
 
 	var/time_left							//Pre-game timer
 	var/start_at
 
-	var/gametime_offset = 432000			//Deciseconds to add to world.time for station time.
```

Comment:
Does this mean round time is in sync with normal time?
If so, is it possible to see how long the round has been running somewhere else?

Reply:
The variable was never used. This will not affect anything.